src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
```

Comment:
yeah, we thought a lot about this when setting up the node-convict utility, but there is probably more that can be done with that - we use `config.{env}.json` files to define project-level config. As @sadafie can attest, it gets complicated very quickly.